src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -205,7 +205,7 @@ trait MemberHandlers {
     val Import(expr, selectors) = imp
     def targetType = intp.global.rootMirror.getModuleIfDefined("" + expr) match {
       case NoSymbol => intp.typeOfExpression("" + expr)
-      case sym      => sym.thisType
+      case sym      => sym.typeSignature
```

Comment:
Also, gah! that didn't do it. Let me do a clean build and upgrade 1.8.0_51 to _60. Desperate times.